src/modules/auth/server/procedures.ts

```
@@ -0,0 +1,151 @@
+                    // Cross Domain Cookies Sharing (if needed)
+                    // sameSite: "none",
+                    // domain: ".yourdomain.com"
+                });
+
+                return {
+                    success: true,
+                    message: "Login successful! ✅",
+                    user: data.user,
+                    token: data.token
+                };
+            } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)